### PR TITLE
Fix test runner for time detection and fix detected time detection bugs

### PR DESCRIPTION
**Pull request by chiragjn**
Replace `[0]?` -> `0?` in regex patterns
Allow "around in" in df type patterns
Allow minutes to be optional in the 24-hour format if "hours?" or "hrs?" is mentioned

### CODE REVIEW

File: ner_v2/detectors/temporal/time/en/time_detection.py [hunk 1]
```diff
@@ -801,7 +802,7 @@ def _detect_24_hour_optional_minutes_format(self, time_list=None, original_list=
             time_list = []
         if original_list is None:
             original_list = []
-        patterns = re.findall(r'\D((00|[0]?[2-9]|[0]?1[0-9]?|2[0-3])[:.\s]([0-5][0-9])?)(?!\s?(?:am|pm|a\.m|p\.m|\d))',
+        patterns = re.findall(r'\b((00?|0?[2-9]|0?1[0-9]?|2[0-3])(?:[:.\s]([0-5][0-9]))?)(?!\s?(?:am|pm|a\.m|p\.m|\d))',
```
Comment (srajdev) [inline ner_v2/detectors/temporal/time/en/time_detection.py]: E501 line too long (120 > 119 characters)

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Comment by haptik-deployment** [comment]
UNIT TESTS FAILED... Please resolve issues

**Comment by chiragjn** [comment]
retest this please

**Comment by haptik-deployment** [comment]
UNIT TESTS HAVE PASSED... Good To Merge

**Pull request merged**